spec/registry.md

```diff
@@ -1109,6 +1109,31 @@ the functions `:datetime` and `:time`:
   - `true`
   - `false`
 
+The following _option_ and its values will be REQUIRED to be available on
```

Comment:
What's the reason to make this option merely recommended?

Reply:
It introduces a dependency on timezone data. Without the option, a `:datetime` formatter doesn't need to know how timezones work, as it can "just" format the input values. With the option, the formatter is expected to know how timezones relate to each other, and when e.g. daylight savings changes happen.

This is particularly relevant as the formatted output often does not include a timezone indicator.

I'm concerned here about the bar we're setting for a minimal valid MF2 implementation, which I don't want to burden with timezone data, esp. given how frequently and unpredictably it changes. Marking this as RECOMMENDED would give a signal to implementers and users that this should not be relied on without checking the capabilities of the MF2 implementation.